azure-cs-functions-linux-app/README.md

```diff
@@ -0,0 +1,66 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# Azure Functions on a Linux App Service Plan
+
+Azure Functions created from deployment packages in Python and deployed to an App Service Plan on Linux.
+
+## Deploying the App
+
+To deploy your infrastructure, follow the below steps.
+
+### Prerequisites
```

Comment:
Also needed to install Python because the function is a Python function?

Reply:
Not really required, as we don't run Python locally - Azure Functions runtime does.

Reply:
I see, no need to compile it locally, as is needed for .NET